### ISSUE TITLE
This library only has sound when playing video?

### ISSUE THREAD

**Issue by Elijah-29** [isamert/empv.el#8]
I played vidoe with `empv-play-video`, but emacs only gave me audio. is this normal？

**Comment by isamert** [comment]
Yes, but you can use `empv-toggle-video` to switch to video mode. Calling it again then will return mpv to audio only mode.  Also please take a look at te following sections of the README:

https://github.com/isamert/empv.el#misc
https://github.com/isamert/empv.el#a-note-about-empv-toggle-video

If you are still having problems feel free to reopen the issue.
